MobileCenter/MobileCenter/Internals/Device/MSDeviceTrackerPrivate.h

```diff
@@ -1,8 +1,8 @@
-#if TARGET_OS_IPHONE && !TARGET_OS_TV
+#if TARGET_OS_IOS
 #import <CoreTelephony/CTCarrier.h>
 #import <CoreTelephony/CTTelephonyNetworkInfo.h>
-#import <UIKit/UIKit.h>
-#elif TARGET_OS_TV
+#endif
+#if TARGET_OS_IPHONE
```

Comment:
TARGET_OS_IPHONE to me feels really confusing as it doesn't only target iPhones :confounded: